### PR TITLE
Fix notification service dependencies linting

### LINKED ISSUE #142: Pydoc issue in ./features/steps/notification_service_dependencies.py:1 at module level:
Missing docstring in public module

### LINKED ISSUE #143: Pydoc issue in ./features/steps/notification_service_dependencies.py:49 in public function `check_token_refreshment_availability`:
First line should end with a period (not 's')

### LINKED ISSUE #144: Pydoc issue in ./features/steps/notification_service_dependencies.py:62 in public function `content_service_is_available`:
Missing docstring in public function

### LINKED ISSUE #145: Pydoc issue in ./features/steps/notification_service_dependencies.py:80 in public function `prom_push_gateway_is_available`:
Missing docstring in public function

### PR DESCRIPTION
**Pull request by JiriPapousek**
# Description

Add docstrings, merge several functions.

Fixes #145, fixes #144, fixes #143, fixes #142

## Type of change

- Refactor (refactoring code, removing useless files)

## Testing steps

Run tests locally.

## Checklist
* [ ] Pylint passes for Python sources
* [ ] sources has been pre-processed by Black
* [ ] updated documentation wherever necessary

### CODE REVIEW

File: features/steps/notification_service_dependencies.py [hunk 1]
```diff
@@ -26,64 +28,69 @@
 
 @when("I retrieve data from insights-content-service on {host:w}:{port:d}")
 @given("insights-content service is available on {host:w}:{port:d}")
-def check_content_service_availability(context, host, port):
+@then("I should get data from insights-content-service")
+def check_content_service_availability(context, host = None, port = None):
     """Check if insights-content-service is available at given address."""
+    assert (host is not None and port is not None) or \
+           (hasattr(context, "content_host") and hasattr(context, "content_port")), \
```
Comment (tisnik) [inline features/steps/notification_service_dependencies.py:35]: test for None is not enough there?

Comment (juandspy) [reply features/steps/notification_service_dependencies.py:35]: yep, I don't get this check. Do we care about `context` having these keys? we are not accessing them in the whole function

Comment (JiriPapousek) [reply features/steps/notification_service_dependencies.py:35]: The context here is for storing host and port if it has been used in one of the previous steps. There is annotation `@then("I should get data from insights-content-service")` (which was originally separate function, that doesn't have host and port). Either this, or I will put those arguments also to this annotation.

Comment (JiriPapousek) [reply features/steps/notification_service_dependencies.py:35]: @juandspy They are accessed on lines 39 and 40.

Comment (tisnik) [reply features/steps/notification_service_dependencies.py:35]: sure, but why not just test for None?

Comment (JiriPapousek) [reply features/steps/notification_service_dependencies.py:35]: Yep, I see it now. Will change that.

**Review by epapbak — APPROVED** [review]
ok =)

**Pull request merged**